apps/settings/js/icc.js

```diff
@@ -500,12 +526,19 @@
       li.appendChild(small);
     }
 
+    if (entry.icon && STKHelper) {
```

Comment:
In order to accomplish this, please follow https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/js/call.js#L3 to make sure all original codes in `icc.js` are wrapped in that anonymous function.